@xen-orchestra/lite/src/stores/vm.store.ts

```diff
@@ -41,16 +45,73 @@ export const useVmStore = defineStore("vm", () => {
       throw new Error(`VM ${id} is halted or host could not be found.`);
     }
 
-    return useXenApiStore().getXapiStats()._getAndUpdateStats({
+    return xenApiStore.getXapiStats()._getAndUpdateStats({
       host,
       uuid: vm.uuid,
       granularity,
     });
   }
 
+  const start = (refs: string[]) =>
+    Promise.all(
+      refs.map((vmRef) => xenApi.value.call("VM.start", [vmRef, false, false]))
```

Comment:
I think we should have a discussion about `xen-api`.

Since it's an API, I would find it more logical to have direct methods.

For example:
- `api.startVm(vmRef)`
- `api.startVmOn(vmRef, hostRef)`
- `api.pauseVm(vmRef)`
- ...

Or:
- `api.vm.start(vmRef)`
- `api.vm.startOn(vmRef, hostRef)`
- `api.vm.pause(vmRef)`
- ...

@MathieuRA  @pdonias @Rajaa-BARHTAOUI What do you think?

Reply:
So instead of having methods exposed by `stores`, directly expose those methods via `xen-api` ?